tests/src/rules/newline-after-import.js

```diff
@@ -164,6 +164,24 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       parserOptions: { sourceType: 'module' },
       parser: 'babel-eslint',
     },
+    {
+      code : `// issue 1004
+        import foo from 'foo';\n
+        @SomeDecorator(foo)
```

Comment:
hm, if a newline is required after `import`, this should be invalid - there needs to be a blank line between the `import` line and the decorator.

Reply:
But at the end of line 169 - '\n', i.e. blank line. Is it not enough?
I think that in this case it is correct, isn't it?